packages/dapp/src/components/SettingsCard/TrustedDappTable.tsx

```diff
@@ -4,23 +4,17 @@ import { TrashIcon } from '@heroicons/react/24/outline';
 import { useTranslations } from 'next-intl';
 
 import { useMascaStore, useToastStore } from '@/stores';
+import Button from '../Button';
+import InputField from '../InputField';
 
-export const FriendlydAppTable = () => {
-  const t = useTranslations('FriendlydAppTable');
+export const TrustedDappTable = () => {
+  const t = useTranslations('TrustedDappTable');
   const [settings, setSettings] = useState<MascaConfig>();
+  const [origin, setOrigin] = useState<string>('https://masca.io');
```

Comment:
Maybe use the current origin, so it will also work on `dev.masca.io` and `beta.masca.io` ?